README.md

```diff
@@ -12,15 +12,22 @@ Compile OOMMF on (Debian) Linux with spack:
 
 [![debian-spack-v0.16.2](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml)
 
-For information: (not part of this `main` branch)
+[![debian-spack-v0.16.3](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml)
+
+
+
+Compile OOMMF on OSX with Spack's `develop` branch (see branch https://github.com/fangohr/oommf-in-spack/tree/osx for these workflows):
 
 - Spack `develop` branch, see https://github.com/fangohr/oommf-in-spack/tree/spack-develop
```

Comment:
Using bullet points here/no bullet points above makes the whole section somewhat inconsistent.

Reply:
The use of bullet points ultimately depends on what exactly is show in the headline above.